Server/src/routes/admin.routes.js

```diff
@@ -7,4 +7,17 @@ const router = express.Router();
 const { validatePath } = validateRoute;
 router.patch('/:userId', checkIsAdmin, validatePath, adminController.changeUserToMentor);
 
+router.get('/:userId', (req, res) => {
+  return res.status(405).json({ status: 405, error: 'Incorrect Method' });
+});
+router.post('/:userId', (req, res) => {
+  return res.status(405).json({ status: 405, error: 'Incorrect Method' });
```

Comment:
You need to get a better way of implementing this, Imagine you had 100 routes would you go doing this?